src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js

```diff
@@ -36,6 +38,7 @@ export function RedirectCreator(props) {
           name="redirectFrom"
           type="text"
           placeholder="URL path to redirect from"
+          value={from}
```

Comment:
If this wasn't present before how was the value showing in the field? What does adding this to the field provide us?

Reply:
I believe the input field was listening on line 45 `evt.target.value` but the input field does not clear after the submit if I dont have line 41 inserted `value={from}`. Please let me know if I am going about this incorrectly thanks.

Reply:
Is there a cursor jump bug now that this is a controlled component?

Reply:
No input is cleared, here is a quick video for both working and non working links
https://www.loom.com/share/6e3cd2fdf67242bfbde8ac34d36e7249

Reply:
@shrunyan there is not a cursor jump on the `From` section but there is in `To` section which is happening currently on master prior to my fixes.

Reply:
Ok. Would you split that bug of into a new ticket?